pennylane/qnode.py

```diff
@@ -186,6 +190,11 @@ def construct(self, args, kwargs=None):
                 however PennyLane does not support differentiating with respect to keyword arguments.
                 Instead, keyword arguments are useful for providing data or 'placeholders'
                 to the quantum circuit function.
+
+        Raises:
+            QuantumFunctionError: if the :class:`pennylane.QNode`'s _current_context is attempted to be modified
+                inside of this method, the quantum function returns incorrect values or if
+                both continuous and discrete operations are specified in the same quantum circuit
```

Comment:
Note that changes here will also need to be duplicated for the new QNode once merged :)

Reply:
Added these to the current ``pennylane/qnode_new/qnode.py``!